.gems

```diff
@@ -1 +1,2 @@
 citrus -v 3.0.2
+minitest -v 5.7.0
```

Comment:
Why do you say that `minitest` is better ?
Honestly I didn't use `minitest` because I wanted to keep simple the `gems dependencies` even for dev.

Anyways, it's ok, I just wonder why did you need it?